### ISSUE TITLE
[Bug]: Voice Over and TalkBack for Building Abbreviations On the Map Does Not Work and Support on the Building Info Sheet Is Fully Missing

### ISSUE THREAD

**Issue by Carlaeatdirt** [elif5446/S390_404-Name-not-Found#150]
### Bug Description

The voice over and talkback for the map elements doesn't work for the buildig initials it only voices out "map pin". The Voice Over and talkback have not been implemented at all for the building info sheet

### Steps to Reproduce

1. Go to phone settings
2. activate voice over
3. run project 
4. open expo go 
5. tap elements on the map 
6. tap the buildings 
7. open the building info sheet
8. tap the elements on the sheet

### Expected Behavior

The voice over should say the names of the buildings when they are tapped example for the hall building it should say "Hall Building". The sheet's elements should be read aloud

### Actual Behavior

Right now the voice over only says "map pin" when a building is tapped and this is the case for both campuses. The sheet's elements are not properly being read

### Environment

OS used for testing: iOS, Android

### Logs / Screenshots

```shell

```

### Confirmation

- [x] I have searched existing issues to make sure this bug has not been reported.
